src/openklant/components/klantinteracties/models.py

```diff
@@ -0,0 +1,338 @@
+import uuid
+
+from django.core.validators import MaxValueValidator
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from vng_api_common.descriptors import GegevensGroepType
+
+from .constants import SoortPartij
+
+
+class Partij(models.Model):
+    id = models.UUIDField(
+        primary_key=True,
+        unique=True,
+        default=uuid.uuid4,
+        max_length=40,
+        help_text=_(
+            "Unieke (technische) identificatiecode van de betrokkene bij klantcontact."
+        ),
+    )
+    nummer = models.PositiveIntegerField(
+        _("Nummer"),
+        help_text=_(
+            "Uniek identificerend nummer dat tijdens communicatie tussen mensen kan "
+            "worden gebruikt om de specifieke partij aan te duiden."
+        ),
+        validators=[MaxValueValidator(9999999999)],
```

Comment:
Interger fields don't have a max_length thats why I used the MaxValueValidator because as you can see the length is documented as 10.

![Screenshot from 2023-10-12 17-47-56](https://github.com/maykinmedia/open-klant/assets/101265650/038ba14e-b862-43ca-a8d1-5a29bb71fee0)

You are correct that they didn't state that it is an interger but because of the description and name it makes the most sense.